recipes/cyvlfeat/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cyvlfeat" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95e63e4451a0368aa26fcc45175b36d3990c271329855ea155b687dc6ca26f57
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+     - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+    - vlfeat
+
+  run:
+    - python
+    - {{ pin_compatible('vlfeat', min_pin='x.x.x', max_pin='x.x.x') }}
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - cyvlfeat.sift.sift
+    - cyvlfeat.sift.dsift
+    - cyvlfeat.fisher.fisher
+    - cyvlfeat.hog.hog
+    - cyvlfeat.kmeans.kmeans
+    - cyvlfeat.generic.generic
+
+about:
+  home: https://github.com/menpo/cyvlfeat
+  license: BSD
```

Which BSD licence is it? From a quick glance I think it's this but please check
```suggestion
  license: BSD-2-Clause
```

---

Thanks